test/TestMain.hx

```diff
@@ -1,5 +1,3 @@
-package ;
```

Comment:
You mean require a `package` when it's optional / empty? I think Flixel generally doesn't omit `package` anywhere...